www/fanm02/sp/reset-password.php

```diff
@@ -0,0 +1,60 @@
+<?php
+require_once 'db/Users.php';
+require_once 'db/Tokens.php';
+require_once 'utils/helpers.php';
+
+$message = null;
+$usersDb = new UsersDB();
+$tokensDb = new TokensDB();
+$error = false;
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    $email = $_POST['email'];
+
+    do { 
+        if(empty($email)){
+            $message = 'Please fill in the email field';
+            $error = true;
+            break;
+        }
+
+        $user = $usersDb->getUser('', $email);
+
+        if($user == null){
+            $message = 'No user was found under this email';
+            $error = true;
+            break;
+        }
+
+        $token = bin2hex(random_bytes(32));
+
+        $tokensDb->create([$email, $token, currentDate('+15 minutes')]);
+
+        $resetLink = "https://eso.vse.cz/~fanm02/sp/new-password.php?token=$token";
```

Comment:
tohle URL by melo byt nekde v configu, co kdyz to budete hostovat na jinem serveru